src/main.rs

```diff
@@ -202,17 +202,32 @@ fn init_log() {
 #[cfg(not(feature = "logging"))]
 fn init_log() {}
 
-fn get_languages(env_lang: Option<&str>, env_language: Option<&str>) -> Vec<String> {
+/// Return language(s) with the following precedence:
+///
+/// 1. CLI language parameter (`-L`, `--language`)
+/// 2. Language as set in the config file
+/// 3. If the `$LANG` env var is set: `$LANGUAGE` (list separated by `:`) followed by `$LANG` (single language)
+/// 4. Fallback: English
```

Comment:
This is not exactly what we want - the "Fallback: English" should only apply to the environment variables, not to the CLI or config file setting. Hence I suggest the following:
- Leave `get_languages` as is, maybe only rename it to `get_languages_from_env` or so and remove the current `get_languages_from_env`
- Make a new function that takes `args` and `config` and tries `args`, then `config` and finally uses `get_languages_from_env`.
- To allow for testing without setting environment variables, the new function should only redirect to another function with a signature like your current `get_languages`:
```rust
fn the_new_function_impl(
    command_line_lang: Option<&str>,
    config_lang: Option<&str>,
    env_lang: Option<&str>,
    env_language: Option<&str>,
) -> Vec<String> {
```